dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -641,8 +644,8 @@ public List<String> getConfiguration() {
     return getOptions().valuesOf(optionConfiguration);
   }
 
-  public Optional<File> getOutputFile() {
-    return optionAsOptional(optionOutput).filter(file -> !file.getName().isEmpty());
+  public Optional<String> getOutputFile() {
+    return optionAsOptional(optionOutput).filter(file -> !"".equals(file));
```

Comment:
`StringUtils#isBlank` or `String#isEmpty` (or some `isNullOrEmpty` variant if file is nullable).

Reply:
Ok